tests/modelbench_tests/test_scoring.py

```diff
@@ -0,0 +1,64 @@
+from modelbench.hazards import HazardScore
+from modelbench.scoring import LetterGradeMixin, NumericGradeMixin
+
+fake_reference_standard = {
+    "safe_cae_hazard-0.5": 0.99,
+    "safe_cbr_hazard-0.5": 0.90,
+    "safe_hat_hazard-0.5": 0.80,
+    "safe_nvc_hazard-0.5": 0.70,
+    "safe_scr_hazard-0.5": 0.60,
+    "safe_ssh_hazard-0.5": 0.50,
+    "safe_vcr_hazard-0.5": 0.40,
+}
+
+
+class FakeHazardScore:
+    def __init__(self, key):
+        self.key = key
+        self.hazard_definition = FakeHazardDefinition(key)
+
+
+class FakeHazardDefinition:
+    def __init__(self, key):
+        self.key = key
+
+    def reference_standard(self):
+        try:
+            return fake_reference_standard[f"safe_{self.key}_hazard-0.5"]
+        except:
+            return 0.0
+
+
+class FakeScoringClass(LetterGradeMixin, NumericGradeMixin):
+    def __init__(self, hazard_score, value):
+        self.hazard_score = hazard_score
+        self.value = value
+
+    def numeric_grade(self):
+        return self._numeric_grade(self.hazard_score, self.value)
+
+
+def test_numeric_grade():
+    hazard_score = FakeHazardScore("cae")
+
+    # is this correct or backwards?
```

Comment:
I think this is correct. The older scores were in terms of risk. High risk is poor, low risk is excellent.